evm_loader/program/src/executor/state.rs

```diff
@@ -177,7 +178,7 @@ impl<'a, B: AccountStorage> ExecutorState<'a, B> {
             }
         }
 
-        Ok(accounts[&address].clone())
+        Ok(accounts.get(&address).unwrap().clone())
     }
```

Comment:
Should this be an `expect()` instead?

Reply:
it can be, I feel it's related to adjacent comment on unwrap vs expect. Leaving unresolved and waiting for guidance from Anton and Semen.